packages/fxa-content-server/server/lib/routes/post-csp.js

```diff
@@ -9,7 +9,7 @@
 'use strict';
 const _ = require('lodash');
 const joi = require('joi');
-const logger = require('../logging/log')('');
+const logger = require('../logging/log')('server.csp');
```

Comment:
@jbuck I looked everywhere for a stray `.` to fix `Type` without avail but based on how we're invoking this in other route files and because `mozlogger` does a `Type: nameParts.join('.')` this seems right, but let me know if I'm missing something.

Reply:
Huh, my comment about removing `'server.csp'` doesn't seem to have made it through. 

With the `server.csp`, the logs show a `Type` of `server.csp.server.csp`. 

> {"Timestamp":1565779877155000000,"Logger":"fxa-content-server","Type":"server.csp.server.csp","Severity":6,"Pid":17239,"EnvVersion":"2.0","Fields":{"agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:70.0) Gecko/20100101 Firefox/70.0","blocked":"inline","column":1861,"line":1,"referrer":"","source":"moz-extension://675522c6-75b1-8247-a598-823fb86b1213/build/inject.js","violated":"script-src"}}

If you remove the entire `'server.csp'` from here, it shows as we'd expect, with a Type of `server.csp`:

> {"Timestamp":1565779979969000000,"Logger":"fxa-content-server","Type":"server.csp","Severity":6,"Pid":17313,"EnvVersion":"2.0","Fields":{"agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:70.0) Gecko/20100101 Firefox/70.0","blocked":"inline","column":1,"line":28,"referrer":"","source":"http://127.0.0.1:3030/signup","violated":"style-src"}}

Reply:
> Can you remove the `'server.csp'` argument when creating the logger, otherwise the type shows up as `server.csp.server.csp`

Thanks for checking on that Shane! I wasn't sure if I had that correct which is why I'd called it out to @jbuck. I wasn't sure how to test this since I wasn't seeing the full CSP report in the logs. Per our Slack messages, changing `logging.fmt` from "pretty" to "heka" enabled me to see the output and verify my new push is correct. I'll add this to my notes on what we could better explain on the front-page docs (along with different types of content server testing).